inference/scripts/build-wasm.py

```diff
@@ -57,41 +57,7 @@ def ensure_docker():
     )
 
 
-def git_clone_update(name: str, repo_path: str, repo_url: str, revision: str):
-    if not os.path.exists(repo_path):
-        print(f"\n⬇️ Clone the {name} repo into {repo_path}\n")
-        subprocess.check_call(
-            ["git", "clone", repo_url],
-            cwd=THIRD_PARTY_PATH,
-        )
-
-    local_head = subprocess.check_output(
-        ["git", "rev-parse", "HEAD"],
-        cwd=repo_path,
-        text=True,
-    ).strip()
-
-    def run(command):
-        return subprocess.check_call(command, cwd=repo_path)
-
-    if local_head != revision:
-        print(f"The head ({local_head}) and revision ({revision}) don't match.")
-        print(f"\n🔎 Fetching revision {revision} from {name}.\n")
-        run(["git", "fetch", "--recurse-submodules", "origin", revision])
-
-        print(f"🛒 Checking out the revision {revision}")
-        run(["git", "checkout", revision])
-        run(["git", "submodule", "update", "--init", "--checkout", "--recursive"])
-
-
-def install_and_activate_emscripten(args: ArgNamespace):
-    git_clone_update(
-        name="emsdk",
-        repo_path=EMSDK_PATH,
-        repo_url="https://github.com/emscripten-core/emsdk.git",
-        revision=EMSDK_REVISION,
-    )
-
+def install_and_activate_emscripten():
```

Comment:
Question: Do we have any check here that makes sure we've done a recursive initiation of git submodules? Seems like a point of failure if we don't.